frontend/pages/documentation.vue

```diff
@@ -57,8 +61,13 @@ useHead({
       <div class="flex-shrink-0 w-56 py-3 hidden md:block h-screen overflow-x-scroll sticky top-14">
         <DocTableOfContents :toc-links="tocLinks" :currently-active-id="currentlyActiveId" class="pb-14" />
       </div>
-      <div class="flex-grow !w-[calc(100%-18rem)] p-4 pt-2 mt-4 bg-white w-full">
-        <ContentRendererMarkdown v-if="documentation" :value="documentation" class="markdown-body w-full" />
+      <div class="flex-grow !w-[calc(100%-18rem)] mt-4 w-full">
+        <div v-if="documentationLabel" class="pb-2">
+          <span class="text-gray-400"> {{ documentationLabel }} </span>
+        </div>
+        <div class="bg-white p-4">
+          <ContentRendererMarkdown v-if="documentation" :value="documentation" class="markdown-body w-full" />
+        </div>
```

Comment:
One quick suggestion to fix the page if the frontend is being run outside of the docker is to remove whitespace (which otherwise makes page look broken)

<img width="1552" alt="Screenshot 2023-07-05 at 17 08 21" src="https://github.com/powerhouse-inc/switchboard-boilerplate/assets/3393626/ecc0b980-aa8d-4cd2-882f-f9f67ac6aa1c">

```suggestion
        <div v-if="documentation" class="bg-white p-4">
          <ContentRendererMarkdown :value="documentation" class="markdown-body w-full" />
        </div>
```

The other suggestion, is to show something like `Follow the instructions in the frontend/README to generate the docs` (but probably only in development mode?)